Core/DISET/private/InnerRPCClient.py

```diff
@@ -5,26 +5,26 @@
 
 from DIRAC.Core.DISET.private.BaseClient import BaseClient
 from DIRAC.Core.Utilities.ReturnValues import S_OK
-
+from DIRAC.Core.Utilities.DErrno import cmpError, ENOAUTH
```

Comment:
Hum, careful with this.
I *think* it should be fine and I solved all the issue, but at the beginning, when developing the options to allow extensions to add their own error code, I remember that importing DErrno inside very low level classes was problematic.
So I think it should work, but to be tested